paper-toast.html

```diff
@@ -230,6 +230,7 @@
          */
         _renderClosed: function() {
           this.classList.remove('paper-toast-open');
+          Polymer.IronOverlayBehaviorImpl._renderClosed.apply(this, arguments);
```

Comment:
Here is too early, because `iron-overlay-behavior` will immediately call `_finishRenderClosed` https://github.com/PolymerElements/iron-overlay-behavior/blob/master/iron-overlay-behavior.html#L325
Instead, you should wait for the animation to be over, and only then call `this._finishRenderedClosed`